playbook-website/app/views/samples/icons.html.erb

```diff
@@ -0,0 +1,172 @@
+<%= pb_rails("flex", props: {justify: 'center'}) do %>
+  <%= pb_rails("flex/flex_item") do %>
+    <%= pb_rails("title", props: {size: 1, text: "Playbook Icons", padding_top: "sm" })%>
+  <% end %>
+<% end %>
+<%= pb_rails("flex", props: {justify: 'center'}) do %>
+  <%= pb_rails("flex/flex_item") do %>
+    <%= pb_rails("title", props: {size: 3, text: "Rails", padding_bottom: "sm"})%>
+  <% end %>
+<% end %>
+
+<%= pb_rails("flex", props: { padding_left: "md"} ) do %>
+  <%= pb_rails("flex/flex_item") do %>
+    <%= pb_rails("title", props: {size: 3, text: "Power Icons", padding_bottom: "sm"})%>
+  <% end %>
+<% end %>
+
+<%= pb_rails("flex", props: { padding_left: "md" }) do %>
+  <%= pb_rails("card", props: { padding: "none", header: true, margin_bottom: "sm"}) do %>
+    <%= pb_rails("card/card_body", props: { padding: "md",  }) do %>
+      <%= pb_rails("icon", props: { icon: image_pack_url("roofing.svg") }) %>
```

Comment:
I've put the svgs in the asset pipeline so that we can achieve the suggested prop style
 `<%= pb_rails("icon", props: { icon: "roofing" }) %>` works now